addons/escoria-core/game/core-scripts/esc/commands/print.gd

```diff
@@ -0,0 +1,26 @@
+# `print string`
```

Comment:
Doesn't `debug` ESC command do the same thing?

Reply:
debug only works if you're in debug log level. I think a better solution is to not use this command but instead allow print to work for all log levels (info/warning etc). I'll remove this PR if you agree.